data/piyush-multiplexer.json

```
@@ -0,0 +1,55 @@
+{
+  "name": "Piyush Goyani",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Full Stack Software Engineer(Majorly Front), Occasionally Python Coder, ML Enthusiastic, Plant Based Programmer | Blogger",
+  "avatar": "https://github.com/SaraJaoude.png",
```

Please update it with your avatar url! 
```suggestion
  "avatar": "https://github.com/piyush-multiplexer.png",
```

---

Added